### PR TITLE
Add the needed support to start using the rust version of the virtiofsd

### PR DESCRIPTION
**Pull request by fidencio**
As the very basic steps on the tests repo, we need to:
* Create a script to install virtiofsd;
* Ensure it gets installed for the supported architectures;

For this we can rely on the work that's been proposed on the kata-containers repo (see: https://github.com/kata-containers/kata-containers/pull/4233) and take advantage of the installation script already provided there (as already done for Cloud Hypervisor).

Fixes: #4784

### CODE REVIEW

File: .ci/install_kata.sh
Comment (c3d) [inline .ci/install_kata.sh:57]: Does that mean we get the Rust version for x86 and C for the other architectures?

Comment (fidencio) [reply .ci/install_kata.sh:57]: Yes, and I've already started the conversation with the maintainers of the other arches to add support for the rust version (see: https://katacontainers.slack.com/archives/C879ACQ00/p1652290987328349?thread_ts=1652288278.597539&cid=C879ACQ00).

Once the x86_64 version gets merged I will open issues with detailed instructions on how to proceed with the other arches.

Comment (fidencio) [reply .ci/install_kata.sh:57]: I could, but that's out of the scope of my work.

I will provide the needed info for others to build and test on the arches they support, but I am not committed to spend time on that.

Fair?

**Review by cmaf — APPROVED** [review]
thanks @fidencio

File: .ci/install_kata.sh
Comment (jodh-intel) [inline .ci/install_kata.sh:57]: Nits:

- You could use a single set of square brackets throughout as I think the only time you need them is for regex matching (`[[ $thing =~ pattern ]]`).
- You could simplify these tests to remove the `if`:
  ```bash
  [ "${arch}" = "x86_64" ] && "${cidir}/install_virtiofsd.sh"
  ```

File: .ci/install_virtiofsd.sh
Comment (jodh-intel) [inline .ci/install_virtiofsd.sh:24]: Nit: I know `tar` can handle this, but for consistency with `-C` I'd consider adding the dash:

```suggestion
	sudo tar -xvJpf build/kata-static-virtiofsd.tar.xz -C /
```

**Review by jodh-intel — APPROVED** [review]
Thanks @fidencio - A few nits but overall...

lgtm

File: .ci/install_kata.sh
Comment (fidencio) [reply .ci/install_kata.sh]: Updated, thanks!

File: .ci/install_virtiofsd.sh
Comment (fidencio) [reply .ci/install_virtiofsd.sh]: Updated here, and also in the `install_cloud_hypervisor.sh` script.

**Comment by fidencio** [comment]
> /test-s390x

Knowing the CI is flaky and it's passed before, I will go ahead and merge this one without waiting for this CI to finish.

**Pull request merged**
